packages/lesswrong/components/LWBackgroundImage.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Components, registerComponent } from '../lib/vulcan-lib';
+import { useLocation } from '../lib/routeUtil';
+import { getReviewPhase } from '../lib/reviewUtils';
+import classNames from 'classnames';
+
+const styles = (theme: ThemeType) => ({
```

Comment:
I think we should default to `defineStyles` and `useStyles`, so that things work slightly better with vite (i.e. preserving local component state with HMR)